components/google_sheets/sources/new-updates/new-updates.js

```diff
@@ -210,17 +215,27 @@ module.exports = {
       return;
     }
 
-    const { files, newPageToken } = await this.google_drive.getModifiedSheets(
+    const { file, newPageToken } = await this.google_drive.getModifiedSheet(
       pageToken,
       this.drive === "myDrive" ? null : this.drive,
-      sheetIDs
+      this.sheetID
     );
-    for (const file of files) {
+    if (file) {
       const spreadsheet = await this.google_sheets.getSpreadsheet(file.id);
       for (const sheet of spreadsheet.sheets) {
-        const { oldValues, currentValues } = await this.getValues(spreadsheet, sheet, file);
+        if (
+          this.worksheetIDs.length > 0 &&
+          !this.worksheetIDs.includes(sheet.properties.sheetId)
```

Comment:
should read `sheet.properties.sheetId.toString()`. In my tests we were `continue`ing every time because `sheetId` was an int but `this.worksheetIDs` contains strings.